cyvcf2/tests/test_reader.py

```diff
@@ -54,6 +54,13 @@ def test_type():
     for v in vcf:
         if len(v.REF) == 1 and len(v.ALT[0]) == 1:
             assert v.var_type == 'snp'
+        if len(v.REF) != 1:
```

Comment:
can you find a specific variant in the VCF, check for e.g. v.REF == "AT" and v.ALT[0] == "CG" and then check v.is_mnp? 
this makes sure the test is executed and avoids replicating logic.

Reply:
can you do this above? this avoids duplicating the logic in the test and indicates exactly which REF/ALT combinations you indicate as MNP

Reply:
I do not get what you mean with above. A MNP is defined as same length of REF and ALT, no matter it ALT is single or multi-allelic. In the test.mnp.vcf there is one variant, which has 4 nucleotides for REF, and either 3 or 4 for ALT, which becomes an indel in that case. So a statement that checks if all ALT alleles are as long as REF should do it. and v.ALT[0] is not enough in that case.

Reply:
And of course a MNP is only if len(REF) > 1

Reply:
I mean the test should be:

```
if (v.REF, v.ALT[0]) in [("AGG","CGA"), (...)]:
    assert var_type == "mnp"
...
```
so we can see exactly what the MNP ref, alt combos are, rather than duplicating the logic.

Reply:
Ok, but these combos need to exist in the test.vcf.gz file, correct?
I did not find any so I created the test.mnp.vcf file, for this one I can set the test cases.

Reply:
yes, just copy out the variants in your new vcf that are mnps